joomla/joomla-cms
# [4.0] Change duplicate label to span for extension name in extensions manager list view

Pull Request for Issue [https://github.com/joomla/joomla-cms/pull/25642#issuecomment-513395666](https://github.com/joomla/joomla-cms/pull/25642#issuecomment-513395666) and [https://github.com/joomla/joomla-cms/pull/25642#issuecomment-513403795](https://github.com/joomla/joomla-cms/pull/25642#issuecomment-513403795).

### Summary of Changes

Change HTML element of the extension name from `<label>` to `<span>` in the extensions manager's list view because the check box to select the item is already a label for the same thing, and duplicate labels for same thing is invalid HTML.

See also the comment mentioned above as issue.

### Testing Instructions

Inspect with your browser's developer tools the HTML of the extensions manager's list view.

Check the label element just before the check box and then the extension name for a particular extension.

Or validate the page and check for duplicate label errors.

### Expected result

Only 1 label for the check box, which is the element before the check box, and the extension name is not another label for that.

### Actual result

1 label for the check box, which is the element before the check box, and the extension name is another label for that.

### Documentation Changes Required

None.
